### ISSUE TITLE
Prompt for stack name format is wrong with local backend

### ISSUE THREAD

**Issue by jen20** [pulumi/pulumi#2761]
When logged into the `local` backend with `pulumi login --local`, `pulumi stack init` prompts for stack names in a format seemingly not supported by the local backend:

```
Please enter your desired stack name.
To create a stack in an organization, use the format <org-name>/<stack-name> (e.g. `acmecorp/dev`): testorg/test-stack
Enter your passphrase to protect config/secrets:
Re-enter your passphrase to confirm:
error: could not create stack: validating stack properties: invalid stack name: a stack name may only contain alphanumeric, hyphens, underscores, or periods
```

**Comment by Frassle** [comment]
Alternatively I don't see any reason the local backend couldn't support orgs and projects? Might be tricky to implement in a way that's backward compatible but surely orgs and projects could just be folders?

Won't be as full featured as the pulumi service but currently we have to tack project names onto all stack names manually.

**Comment by mikhailshilkov** [comment]
This has been fixed in #3066 and then further improved in #3205
